pmd-java/src/main/resources/category/java/errorprone.xml

```diff
@@ -2180,12 +2180,21 @@ public class Foo extends TestCase {
           language="java"
           since="1.5"
           message="Classes should not have non-constructor methods with the same name as the class"
```

Comment:
```suggestion
          message="A method should not have the same name as its containing class"
```

"Non-constructor method" is nonsense, and the violation is reported on the method, not the class.